Assets/Scripts/Source/Game/Combat/CharacterAction.cs

```diff
@@ -23,6 +23,14 @@ public CharacterAction(Character actor, Action onUseHandler = null, params Chara
         this.Targets = targets;
     }
 
+    public virtual void AddBonus()
```

Comment:
Nit: perhaps add a comment that these are "no-op" functions and should be overridden in subclass.

Reply:
What's a no-op.